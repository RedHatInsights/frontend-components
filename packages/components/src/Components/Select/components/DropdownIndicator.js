import React from 'react';
import { CaretDownIcon } from '@patternfly/react-icons';

const DropdownIndicator = () => <CaretDownIcon />;

export default DropdownIndicator;
