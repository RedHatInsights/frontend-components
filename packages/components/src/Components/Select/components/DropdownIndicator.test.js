import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DropdownIndicator from './DropdownIndicator';

describe('DropdownIndicator component', () => {
    describe('should render', () => {
        it('required props', () => {
            const wrapper = shallow(<DropdownIndicator />);
            expect(toJson(wrapper)).toMatchSnapshot();
        });
    });
});
