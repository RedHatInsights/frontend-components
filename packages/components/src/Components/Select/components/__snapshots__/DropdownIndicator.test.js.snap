// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownIndicator component should render required props 1`] = `
<CaretDownIcon
  color="currentColor"
  size="sm"
  title={null}
/>
`;
